osf/models/user.py

```diff
@@ -504,6 +504,12 @@ def csl_name(self, node_id=None):
                 'given': csl_given_name,
             }
 
+    @property
+    def osfstorage_region(self):
+        from addons.osfstorage.models import Region
+        user_settings = self.get_addon('osfstorage')
+        return Region.objects.get(id=user_settings.default_region_id)
```

Comment:
You can do this in one query:

```python
default_region_subquery = UserSettings.objects.filter(owner=me.id).values('default_region_id')
Region.objects.get(id=default_region_subquery)
```

Reply:
FYI the property ends up looking like this since `addons.osfstorage.models.UserSettings` is the only `UserSettings` model with a concept of `default_region`.
```
@property
def osfstorage_region(self):
    from addons.osfstorage.models import Region
    osfs_settings = self._settings_model('osfstorage')
    default_region_subquery = osfs_settings.objects.filter(owner=self.id).values('default_region_id')
    return Region.objects.get(id=default_region_subquery)
```